vtex-apps/store-components
# Add showValueForVariation prop to SKUSelector

#### What problem is this solving?

Add the `showValueForVariation` prop to the `SKUSelector`

#### How should this be manually tested?

[Workspace](https://skuselector--storecomponents.myvtex.com/classic-shoes/p?skuId=35)

There is this workspace `skuselectortest` and you can see this change applied to these accounts:

```json
[
  "storecomponents",
  "exitocol",
  "boticario",
  "alssports",
  "melimeloparis",
  "parquedpedro",
  "samsungar",
  "tokstoklista",
  "ecowaterqa",
  "samsungempresarial",
  "lesters",
  "btglobal",
  "samsungpeepp",
  "spaziolusso",
  "naturitasit",
  "lily",
  "tbb",
  "worldwidegolf",
  "motorolauk",
  "gc-jqu0734",
  "garmin",
  "paguemenos",
  "carulla"
]
```

#### Checklist/Reminders


#### Screenshots or example usage



#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes
